creator-node/src/app.js

```diff
@@ -64,6 +67,8 @@ const initializeApp = (port, serviceRegistry) => {
   // https://expressjs.com/en/guide/behind-proxies.html
   app.set('trust proxy', true)
 
+  app.set('serviceRegistry', serviceRegistry)
```

Comment:
NOTE - rather than setting each individual property on the app object, thought it would be simpler to just set the full registry and each request can pick off services as needed. this is additive only but was thinking moving fwd can use this pattern like i do in replicaSetController.js, open to feedback here tho